docs/src/modules/components/DemoSandboxed.js

```diff
@@ -10,10 +10,10 @@ import DemoErrorBoundary from 'docs/src/modules/components/DemoErrorBoundary';
 function FramedDemo(props) {
   const { children, document } = props;
 
-  const theme = useTheme();
+  const { direction } = useTheme();
```

Comment:
I think that we should try to retain the `theme.x` structure as much as possible in the codebase. The incentive for not using such approach is  that it makes it harder to grep in the codebase. Grep can be used to studying usage or refactoring.

Reply:
It's invalid code.

Reply:
@dependabot squash and merge

Reply:
> you're suggesting to not use destructuring at all

For the theme, yes, exactly, because it prevents to study the codebase with simple queries like `theme.palette`, etc.

> It's invalid code

Why is it invalid?